app/src/main/java/com/peacecorps/pcsa/safety_resources/SafetyResourcesFragment.java

```diff
@@ -8,16 +8,22 @@
 
 import com.peacecorps.pcsa.R;
 
-
-/**
- * A placeholder fragment containing a simple view.
+/*
+ * Safety Resources main fragment 
+ *
+ * @author calistus
+ * @since 2015-08-18
  */
 public class SafetyResourcesFragment extends Fragment {
 
     public SafetyResourcesFragment() {
     }
-
-    @Override
+    /**
+     * A placeholder fragment containing a simple view 
+     * 
+     * onCreateView method is called when Fragment should create its View object hierarchy
```

Comment:
@return and @param comments are missing. See yogeshbalan's last PR which was merged regarding this. Follow the template given